osgp/protocol-adapter-oslp/osgp-adapter-protocol-oslp-elster/src/main/java/org/opensmartgridplatform/adapter/protocol/oslp/elster/infra/messaging/DeviceRequestMessageProcessor.java

```diff
@@ -139,11 +137,12 @@ protected void handleScheduledEmptyDeviceResponse(final DeviceResponse deviceRes
         responseMessageSender.send(responseMessage);
     }
 
-    protected void handleError(final Exception e, final String correlationUid, final String organisationIdentification,
-            final String deviceIdentification, final String domain, final String domainVersion,
-            final String messageType, final int messagePriority, final int retryCount) {
+    protected void handleError(final IOException e, final String correlationUid,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 9 parameters, which is greater than 6 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS00107)